Xamarin.Forms.Platform.UAP/Platform.cs

```diff
@@ -530,6 +533,45 @@ static void OnPageActionSheet(object sender, ActionSheetArguments options)
 			}
 		}
 
+		static async void OnPagePrompt(Page sender, PromptArguments options)
+		{
+			var promptDialog = new PromptDialog
+			{
+				Title = options.Title ?? string.Empty,
+				Message = options.Message ?? string.Empty,
+				Input = options.InitialValue ?? string.Empty,
```

Comment:
@jfversluis Yep, should be right here.  It's also shown in the third screen shot

Reply:
🤦‍♂️ great stuff! Thanks!